client-java/src/main/java/uk/gov/gchq/palisade/client/Client.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020-2021 Crown Copyright
```

Comment:
Replace copyright template with Copyright 2018-2021 Crown Copyright for all Palisade files